### ISSUE TITLE
previously valid config started to fail kube-aws validate

### ISSUE THREAD

**Issue by jcrugzz** [kubernetes-incubator/kube-aws#1221]
Without changing the config at all. After running into the etcd issues being discussed in #1206, in attempting to debug starting up a cluster I started to get a validation error:

```
Validating UserData and stack template...
Error: failed to validate node pool #0: create volume dry-run request failed: InvalidParameterCombination: The parameter iops is not supported for gp2 volumes.
        status code: 400, request id: efb84700-265a-4e8e-8e5b-fc5922d688b1
```

The odd part about this is that I did not modify any defaults when it comes to volumes, I havent even enabled anything to do with volumes explicitly. Is there something new on aws that doesnt like the default configuration?

**Comment by ktateish** [comment]
Hi @jcrugzz 
It looks like #1170 which is fixed by #1171 .

**Comment by jcrugzz** [comment]
@ktateish totally missed that issue, thanks!
